Vagrantfile

```diff
@@ -17,6 +17,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box_version = "> 0.3.2.1714"
 
   config.vm.provision :shell, :inline => $bootstrap, :privileged => true
+  # Use the git configuration from the host in the VM
+  config.vm.provision "file", source: "~/.gitconfig", destination: ".gitconfig"
```

Comment:
We should take a step back and look at the bigger issue: we're using this image for two things, developer-facing test box and as part of release process.

Insofar as this particular piece of functionality is irrelevant or detrimental to first use case, perhaps it should be done in way that is specific to second use case only. Which is to say, not in the Vagrantfile. Or perhaps we want separate Vagrantfiles for the different use case.